protocol/avdl/secret_ui.avdl

```diff
@@ -37,13 +37,15 @@ protocol secretUi {
 
   string getPaperKeyPassphrase(int sessionID, string username);
 
-  record SecretStorageFeature {
+  record Feature {
     boolean allow;
+    boolean respond; /* if we want the value back */
     string label;
   }
 
   record GUIEntryFeatures {
-    SecretStorageFeature secretStorage;
+    Feature storeSecret;
+    Feature showTyping;
```

Comment:
When would showTyping.Allow ever equal false?  And when would the caller ever want to set showTyping.Label?